src/extension/wmts-multi-dimensional/src/main/java/org/geoserver/gwc/wmts/dimensions/Dimension.java

```diff
@@ -5,15 +5,7 @@
 package org.geoserver.gwc.wmts.dimensions;
 
 import java.io.IOException;
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.Date;
-import java.util.List;
-import java.util.Map;
-import java.util.Optional;
-import java.util.Set;
-import java.util.TreeMap;
-import java.util.TreeSet;
+import java.util.*;
```

Comment:
avoid wildcard import. This will also make the qa build fail. you can disable it from intellij.